lib/bpmn/element.rb

```diff
@@ -16,7 +17,7 @@ def self.from_moddle(moddle)
         klass = "Bpmn::#{moddle["$type"].split(':').last}".constantize
         return klass.new(moddle)
       rescue # => e
-        ap "Create class for #{moddle["$type"]}"
+        ap "Error creating instance of #{moddle["$type"]}"
         return Element.new(moddle)
```

Comment:
I realize it's not part of this PR, but it might be nice to make an obvious callout that an element was unknown, so perhaps in this scenario it's `UnknownElement`, which is just a subclass of `Element`.